### PR TITLE
Simplified the hello-world exercise

### PR DESCRIPTION
**Pull request by KamilMalek**
As per #333

### CODE REVIEW

File: exercises/hello-world/HINTS.md [hunk 1]
```diff
@@ -14,12 +14,11 @@ but it helps to understand what a test looks like, and what
 it is doing.
 
 Open up the test file, hello-world.spec.js.
-It has three tests defined in it.
 
 This is the first test:
```
Comment (matthewmorgan) [inline exercises/hello-world/HINTS.md]: Perhaps this should be changed to say 'There is one test:' or something of that nature?

File: exercises/hello-world/HINTS.md [hunk 2]
```diff
@@ -30,23 +29,23 @@ The test fails, which makes sense since you've not written any code yet.
...
 Now look at the test definition again:
 
-    it('says hello world with no name', function() {
+    it('says hello world', function() {
       // ... more code here ...
     });
 
-The text 'says hello world with no name' is repeated.
+The text 'says hello world' is repeated.
 This is how you know which test failed.
```
Comment (matthewmorgan) [inline exercises/hello-world/HINTS.md]: Again, since there's only one test now, this should be changed to 'This is how you know the test failed.'

**Comment by KamilMalek** [comment]
Does the latest commit take care of it properly?

**Comment by matthewmorgan** [comment]
🥇 Thanks!

**Pull request merged**